### PR TITLE
Added SVGPage component, added to page switch. Added helpers to utils…

### PR DESCRIPTION
**Pull request by thmasjs**
… area. Added documentation.

# Changes

*Describe the changes made, including any changes to tests, dependencies or docs. Link to any relevant issues below.*

- Closes #XXXX *(replace XXXX with GitHub issue number)*
- ...

# Tasks

- [ ] Have unittests been added (testing individual functions and their edge cases)
- [ ] Has documentation (including user guide) been updated
- [ ] Have new dependencies been added (or changed) in relevant `requirements.txt`
- [ ] Code linting, tests and other workflows pass

### CODE REVIEW

**Comment by thmasjs** [comment]
Everything is covered in the .md which has been commited. To be tested with: https://github.com/Transport-for-the-North/Strategic-Analysis-Visualisation-Framework/pull/109

File: packages/vis-core/src/utils/svgPageHelpers.js [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  return false;
+};
+
+/**
+ * Resolve SVG URL from a data row.
+ * Scans multiple column names for raw SVG markup or URLs.
+ * @param {Object} row - The data row containing SVG information.
+ * @returns {string|null} - Data URI or URL pointing to the SVG, or null if not found.
+ */
+export const resolveSvgUrl = (row) => {
```
Comment (brereh) [inline packages/vis-core/src/utils/svgPageHelpers.js]: This is good and what we want to see here. 

TODO: There's potentially other image rendering functionality elsewhere that we might want to combine it with (I think in the CalloutCard). Let's consider that for a future change.
